client/public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <script src="https://kit.fontawesome.com/09fc819f23.js" crossorigin="anonymous"></script>
```

Comment:
generally I'd recommend using the npm package over the CDN script. The reason being, is if the fontawesome site has an outage, fonts would not load in our app.